platforms/posix/src/px4/common/px4-alias.sh_in

```diff
@@ -28,6 +28,19 @@ sh() {
 	. "$(pwd)$script"
 }
 
+source() {
```

Comment:
This comment hints already that overriding `set` and `source` is not a good idea:
https://github.com/PX4/Firmware/blob/2128679ab3e4ba888cc3f8b1cb9b00a398847cc5/platforms/posix/src/px4/common/px4-alias.sh_in#L7

Reply:
The main difficulty is that both `source` and `.` require an absolute path.
If we need to avoid overriding altogether, we can add a prefix variable, `$PWD` on POSIX, and `/` on NuttX: `. ${R}etc/init.d/rc.fw_defaults`

Reply:
@dagar argued we should clean all the invocations up and only uses `sh` hence the action listed in the title block `after this comes in` but maybe we need an interim fix:  

Use $sh and assign to `source` on linux and `.` on nuttx? 

Can we discuss it on the dev-call Wed?

Reply:
I would do whatever works for now and once this settles I'll take a look to see if we can start isolating some of the scripts and not have to track setting and unsetting every variable in rcS.